view/theme/frio/templates/generic_links_widget.tpl

```diff
@@ -0,0 +1,12 @@
+
+<div class="widget{{if $class}} {{$class}}{{/if}}">
+	{{if $title}}<h3>{{$title}}</h3>{{/if}}
+	{{if $desc}}<div class="desc">{{$desc}}</div>{{/if}}
+
+	<ul role="menu">
+		{{foreach $items as $item}}
+			<li role="menuitem" class="{{if $item.selected}}selected{{/if}}"><a href="{{$item.url}}" {{if $item.accesskey}}accesskey="{{$item.accesskey}}"{{/if}}>{{$item.label}}</a></li>
```

Comment:
I usually don't give a proverbial frack on how output from html templates is, given that the source is easy to understand. Browser's dev tool format it, when I need to look at it.
I usually also write forloops in template like this, because the li is in loop and the loop is in ul.. easy to see, easy to find any li eventually out of the loop..

anyway, I've reduced the indentation here and made indentation in the next loop equal to this 👍